client/src/app/core/translate/marked-translations.ts

```diff
@@ -382,3 +382,4 @@ _('Motion change recommendation deleted');
 
 // core misc strings
 _('items per page');
+_('Tag');
```

Comment:
"Hide referring motions" was never marked as a translation in the client and thus does not appear in the template-en.pot after running `npm run extract`